### PR TITLE
Unarmed weapon balance (Part 1 of melee weapon balance project)

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary

```SUMMARY: Balance "Rebalances unarmed weapons"```

#### Purpose of change
Mlangsdorf has been doing some deep analysis of the melee weapons and both analysis of dps of weapons and evaluating their strength ingame, and started a large "melee weapon balance" project. His analysis and his proposed values can be found here:
https://docs.google.com/spreadsheets/d/14eQIe4AO_f6OxCt1XcB4NLAs6-5R1wQW-ydZG0orYdY/edit#gid=1655515751
He wanted someone to start implementing the new values ingame, so i started work.

#### Describe the solution
Rebalances unarmed weapons according to excel sheet. Small adjustments to "tohit bonus" to fit game_balance.MD

#### Additional context
Credit to Mlangsdorf for the excellent analysis and for most of the proposed values.
Credit to vastin for review of the excel sheet.
Credit to hymore24/hirmuolio for the initial complaint.

### CODE REVIEW

**Comment by jbytheway** [comment]
We've been discussing that when large rebalance projects like this happen with spreadsheets or other data to support them it would be really helpful for the data to be recorded in the repo for future reference.  It's not always easy to trace back the changes to their corresponding PRs, and sometimes documents hosted elsewhere disappear or become inaccessible.  This weapon balancing effort seems like an excellent one to set an example in this regard.

I suggest creating a new `doc/reference` for this type of information.  So, for this project: create a subdirectory called e.g. `doc/reference/weapon_balance`, dump all the spreadsheet data there (I suggest using [ods format](https://en.wikipedia.org/wiki/OpenDocument)) and add a README explaining what the approach was, how the formulae were derived, etc. (or at least a pointer to where such details can be found in the spreadsheets).

**Comment by Brian-Otten** [comment]
Sounds great, i'll get in touch with mlangsdorf to see if he can setup such a document (since the excel sheet is his, i'm pretty much doing data entry on these).
@mlangsdorf 
If he asks me to set up something i'll see what can be done.

**Comment by mlangsdorf** [comment]
Spreadsheet and documentation added as part of #39564

**Comment by SirPendrak** [comment]
Not so long ago Kevin nerfed cestus in #36012 
This PR makes it super strong again. 
Probably better to leave it weak cuz i suspect it will meet banhammer again. 
I guess for pure damage you have steel knuckles anyway?

**Comment by mlangsdorf** [comment]
Hmm, that change landed after I had pulled the data.

Revised fist load weapons:

Item Name | Item ID | OVal | Acc | Moves | Bash | Cut | Stab | NVal |   | Acc | Moves | Bash | Cut | Stab | NVal
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
bionic claws | bio_claws_weapon | 40.8 | 4 | 73 | 8 | 0 | 16 | 24.82 |   | -1 | 73 | 8 | 0 | 20 | 17.74
cestus | cestus | 11.9 | 0 | 89 | 2 | 0 | 0 | 2.67 |   | 0 | 89 | 2 | 0 | 0 | 2.67
pair of brass knuckles | knuckle_brass | 10.8 | 0 | 70 | 6 | 0 | 0 | 6.18 |   | 0 | 70 | 3 | 0 | 0 | 4.09
pair of nail knuckles | knuckle_nail | 15.1 | 0 | 76 | 4 | 6 | 0 | 6.78 |   | 0 | 76 | 3 | 4 | 0 | 5.30
pair of steel knuckles | knuckle_steel | 11.5 | 0 | 76 | 8 | 0 | 0 | 6.97 |   | 0 | 76 | 3 | 0 | 0 | 3.77
punch dagger | punch_dagger | 23.6 | 2 | 71 | 2 | 0 | 10 | 11.38 |   | -1 | 71 | 2 | 0 | 14 | 10.22
razorbar katar | knuckle_katar | 25.2 | 2 | 92 | 0 | 16 | 0 | 12.00 |   | 0 | 92 | 3 | 14 | 0 | 9.45
tiger claws | bagh_nakha | 24.0 | 3 | 66 | 1 | 6 | 0 | 8.50 |   | 0 | 66 | 2 | 15 | 0 | 13.23

That makes them all suck, but at least the suck in line with bionic claws.

File: data/json/items/melee/fake.json [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
     "description": "Short and sharp claws made from a high-tech metal.",
-    "to_hit": 4,
+    "to_hit": -1,
     "color": "light_gray",
     "symbol": "{",
     "material": [ "superalloy" ],
     "volume": "500 ml",
     "bashing": 8,
-    "cutting": 16,
+    "cutting": 20,
```
Comment (kevingranade) [inline data/json/items/melee/fake.json:13]:
```suggestion
    "cutting": 16,
```

**Comment by kevingranade** [comment]
The only one I have a super strong opinion about is the cestus, because AFAICT people end up treating it as being a very different thing than what history and the crafting recipe say it is, which is thin metal plates held in place over the knuckles with leather wraps.

**Comment by Brian-Otten** [comment]
Updated to new values by mlangsdorf

**Pull request merged**